### DISCUSSION TITLE
"Content-Type: text/plain; charset=us-ascii" showing up in emails - SOLVED

### DISCUSSION THREAD

**Discussion by zagarskas** [PHPMailer/PHPMailer#2780] [General]
This post is simply here for anyone googling: "Content-Type: text/plain; charset=us-ascii showing up in emails"

Here is the deal:

1. PHPMailer Version 5.0 worked fine up to PHP 7.x
2. Switching to PHP 8.x will cause the following to show up in emails
3. Upgrading to PHPMailer Version 5.5 fixes the problem

Examples of undesired outcomes in emails sent with PHPMailer Version 5.0 running on PHP 8x

```
--b1_000000000000000000000000000
Content-Type: text/plain; charset=us-ascii
_[your plain text email message here]_

--b1_000000000000000000000000000
Content-Type: text/html; charset=us-ascii
_[your html email message here]_
```

**Comment by Synchro** [comment]
Updating is a good idea anyway, but nobody should be running PHPMailer 5.x any more, especially if they have updated to PHP 8.
